sechub-pds-solutions/iac/10-create-image.sh

```diff
@@ -44,9 +44,9 @@ fi
 BUILD_ARGS="--build-arg BASE_IMAGE=$BASE_IMAGE"
 echo ">> Base image: $BASE_IMAGE"
 
-if [[ ! -z "$IAC_VERSION" ]] ; then
-    echo ">> IaC version: $IAC_VERSION"
-    BUILD_ARGS="$BUILD_ARGS --build-arg IAC_VERSION=$IAC_VERSION"
+if [ -n "$KICS_VERSION" ] ; then
```

Comment:
Suggestion: `source ./env` if $KICS_VERSION is empty so the version is always set.